### PR TITLE
UPSTREAM: 46265: Denote if a printer is generic

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin/issues/17163

Picks https://github.com/kubernetes/kubernetes/pull/46265/files
in order to avoid the panic seen in https://github.com/openshift/origin/issues/17163

cc @openshift/cli-review @smarterclayton

### CODE REVIEW

**Comment by smarterclayton** [comment]
This is a lot of change to be picking right at the end of the release.  Is there a simpler fix?

File: pkg/cmd/admin/migrate/migrator.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/resource"
 
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"k8s.io/kubernetes/pkg/printers"
```
Comment (stevekuznetsov) [inline pkg/cmd/admin/migrate/migrator.go]: bad import order

**Comment by juanvallejo** [comment]
@smarterclayton 
> This is a lot of change to be picking right at the end of the release. Is there a simpler fix?

Opened a simpler fix for this here: https://github.com/openshift/origin/pull/17227

**Comment by juanvallejo** [comment]
Closing this in favor of https://github.com/openshift/origin/pull/17227

**Pull request closed without merging**